concordia/settings_dev.py

```diff
@@ -17,12 +17,6 @@
 
 ALLOWED_HOSTS = ["127.0.0.1", "0.0.0.0", "*"]
 
-CELERY_HOST = "localhost"
```

Comment:
Were these settings causing a problem? If you try to run celery locally (a la `pipenv run celery -A concordia worker -l info`) then these need to be in there. The docker environment uses `settings_docker`.

Reply:
Yes, they broke running under docker-compose and the defaults in settings_template are the same.

Reply:
Ok - I just need to add more stuff to my env then.

Reply:
Actually the `settings_dev` redis port should be 63791 (assuming you're running the DB and redis in docker) and in `settings_template` it's 6379.

Reply:
I guess I'm also a bit confused how these settings broke running under docker-compose, since that uses `settings_docker` and not `settings_dev`

Reply:
63791 is the exposed public port but it wasn't available to containers running the network compose created. I changed it to link the containers and that also removed the need to use the non-standard port so the default 6379 worked.

Reply:
It works when running everything as docker, but not when only running redis and db in docker, and django / celery / celerybeat not in docker.

Reply:
I was meaning to ask whether we see enough value from the non-standard port to be worth needing to customize it. For running the apps outside of Docker, I end up needing a custom settings file anyway so I have `settings_dev_cadams.py` which can override things as needed.

Reply:
That works for me